### PR TITLE
Sprint2-Routes-LR: Add basic user login and registration routes 

### LINKED ISSUE #1: Create API routes
Create user login and register routes

### PR DESCRIPTION
**Pull request by Leonardo-Reyes-Munoz**
This feature branch updates the app.js module with a user's route which connect to the userRoutes.js module. Two end points are defined within the users route. A 'register' endpoint will connect to the appropriate controller to allow the user to register. A 'loginUser' end point will allow the API to authenticate the user and provide an authentication token.

### CODE REVIEW

**Review by rebs1203 — APPROVED** [review]
Looks very good so far! Good job!

File: src/controllers/userControllers.js
Comment (shreeup) [inline src/controllers/userControllers.js:1]: may be we can include try catch in controller, if it is part of another task fel free to ignore my comment
we can also send status code in response
like this
`res.status(StatusCodes.OK).send("")`

Comment (shreeup) [reply src/controllers/userControllers.js:1]: Other than that changes look good to me .. i think we can go ahead and merge

**Review by shreeup — APPROVED** [review]

File: src/controllers/userControllers.js
Comment (Leonardo-Reyes-Munoz) [reply src/controllers/userControllers.js:1]: Yes, we can add some of those features once we implement the controller logic in the next pull request.

**Pull request merged**